### PR TITLE
 Fixed issue 2704 by removing the unnecessary notification below password in the Login Panel

### LINKED ISSUE #2704: Unnecessary notification below password in the Login Panel
**Actual Behaviour**
In the Login panel, below password, it show the error notification -"Password should be at least 6 to 64 characters",which is totally unnecessary because user have to input the exact password that he/she used during signup.

**Expected Behaviour**

The following password error notification should be removed 

**Screenshots of the issue**
pass_Comm

**Would you like to work on the issue?**
Yes ,i would like to work on it

### PR DESCRIPTION
**Pull request by tanujvyas10**
Fixes #2704

**Changes: Fixed issue 2704 by removing the unnecessary notification below password in the Login Panel**

**Screenshot**
login

**Demo link**:
[https://young-attack.surge.sh/](https://young-attack.surge.sh/)

### CODE REVIEW

**Comment by tanujvyas10** [comment]
@akshatnitd and @fragm3 .I have made the necessary changes.Please review my work.Thank you

File: src/components/Auth/Login/Login.react.js [hunk 1]
```diff
@@ -159,13 +159,9 @@ class Login extends Component {
       }
       case 'password': {
         const password = event.target.value.trim();
-        let passwordErrorMessage = '';
-        if (!password || password.length < 6) {
-          passwordErrorMessage = 'Password should be atleast 6 characters';
```
Comment (akshatnitd) [inline src/components/Auth/Login/Login.react.js]: We should still show, `Password cannot be empty` message.

**Comment by akshatnitd** [comment]
Just for info, the website is auto-deployed to below mentioned URL after successful build completion.

Link -  https://pr-`<PR-no>`-fossasia-susi-web-chat.surge.sh
For this PR,  https://pr-2764-fossasia-susi-web-chat.surge.sh

File: src/components/Auth/Login/Login.react.js (hunk 1 above)
Comment (tanujvyas10) [reply src/components/Auth/Login/Login.react.js]: sir its for the login panel.I have also attached the surge link .Please check it out

Comment (tanujvyas10) [reply src/components/Auth/Login/Login.react.js]: Sir,if password is empty while login ,the login button will not enable.You may check

Comment (akshatnitd) [reply src/components/Auth/Login/Login.react.js]: @tanujvyas10 We still want to show. User may not be able to understand the reason for the disabled button. Having empty check and showing message is a standard practice.

Comment (tanujvyas10) [reply src/components/Auth/Login/Login.react.js]: In that case, we can permanently enable the login button, and if user click **login** without entering the email or password or both,the user will get the notification below each of the inputs accordingly.Can we do that?

**Comment by tanujvyas10** [comment]
> We require client-side validation, making this PR unwanted. Please close it and pick other issues from repo.

sir, you can see any login form, none of them will tell the user while entering password -**Password should be atleast 6 to 64 character**,it may be valid for signup page.
For the login-page , we can just notify the user to enter the email or password(when left blank).If you want , i can work on that.

**Comment by tanujvyas10** [comment]
> > We require client-side validation, making this PR unwanted. Please close it and pick other issues from repo.
> 
> sir, you can see any login form, none of them will tell the user while entering password -**Password should be at least 6 character**,it may be valid for signup page.
> For the login-page , we can just notify the user to enter the email or password(when left blank).If you want , i can work on that.
> logtosisi

**Comment by akshatnitd** [comment]
IMO, not required. There are over 70 issues open. Feel free to pick some other issue.

**Pull request closed without merging**